apps/backend/db_patches/0168_XpressEmailSetting.sql

```diff
@@ -0,0 +1,12 @@
+DO
+$$
+BEGIN
+  IF register_patch('0168_XpressEmailSetting.sql', 'Chi Kai Lam', 'Xpress email setting for recipients SAMPLE_SAFETY_EMAIL and TECHNIQUE_SCIENTISTS_EMAIL', '2025-01-13') THEN
+    BEGIN
+      INSERT INTO settings (settings_id, settings_value, description) VALUES ('SAMPLE_SAFETY_EMAIL', 'ISISSAMPLESAFETY@stfc.ac.uk', 'Email address for the sample safety.'); 
+      INSERT INTO settings (settings_id, settings_value, description) VALUES ('TECHNIQUE_SCIENTISTS_EMAIL', 'dummy@stfc.ac.uk; test@stfc.ac.uk', 'Email address for the Xpress technique scientist.');
```

Comment:
Can the defaults be left as an empty string or some other value? I don't think the other facilities would want to be setting stfc email addresses, and I don't think our sample safety team would want to recieve emails from other facilites if the settings weren't updated.